packages/editor/packages/history/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "history",
+  "private": true,
```

Comment:
```suggestion
```

Reply:
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#private

Reply:
It depends on how we're gonna publish it (as multiple packages or as a whole "Editor" pacakge):

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#private